tests/test_pytypes.cpp

```diff
@@ -75,6 +75,19 @@ TEST_SUBMODULE(pytypes, m) {
         return dict.contains(val);
     });
 
+    // test_tuple
+    m.def("make_tuple", []() { return py::make_tuple(42, py::none(), "spam"); });
```

Comment:
Isn't this tested anywhere else!? If so, wow, good catch! If not: do we want to repeat this here?

Reply:
There are other tests that rely on `make_tuple` to work, so if it wouldn't work that would not go unnoticed. However, nowhere are specific tests for `make_tuple` grouped. I thought it would be good to be a bit more explicit in that regard and, if nothing else, provide a space for such tests to be collected.